HW_7/logger.py

```diff
@@ -0,0 +1,10 @@
+from datetime import datetime
+
+
+def log(func):
+    def wapper(*args):
+        with open('log.csv', 'a', endecodind='utf-8') as log_file:
+            log_file.write(f'{datetime.now()} - запущена функция {func.__nume__} {"с аргументами" if args else "без аргументов"} ({func.__doc__})\n')
```

Comment:
У меня тут было не дописано. У вас тут тоже ошибка в неправильном имени атрибута `__name__`
```suggestion
            log_file.write(f'{datetime.now()} - запущена функция {func.__name__} {f"с аргументами: {args}" if args else "без аргументов"} ({func.__doc__})\n')
```